### PR TITLE
Added Command Add image to Excel Cell

### PR DESCRIPTION
**Pull request by openbots-aw**
- Added AddImageToExcelCell Command

### CODE REVIEW

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+		[SampleUsage(@"C:\temp\image.png || {vFilePath} || {ProjectPath}\image.png")]
+		[Remarks("")]
+		[Editor("ShowVariableHelper", typeof(UIAdditionalHelperType))]
+		[Editor("ShowFileSelectionHelper", typeof(UIAdditionalHelperType))]
+		[CompatibleTypes(null, true)]
+		public string v_ImagePath { get; set; }
+
+		public ExcelAddImageToCell()
+		{
+			CommandName = "ExcelAddImageToCell";
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs:53]: Please modify the all instances of "Add Image To Cell" to "Write Image To Cell"

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+using System.Windows.Forms;
+using Application = Microsoft.Office.Interop.Excel.Application;
+using Microsoft.Office.Core;
+using Microsoft.Office.Interop.Excel;
+
+namespace OpenBots.Commands.Excel
+{
+    [Serializable]
+    [Category("Excel Commands")]
+    [Description("This command adds an image to excel cell.")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs:21]:
```suggestion
    [Description("This command writes an image to a specific cell in an Excel Worksheet.")]
```

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+		[Required]
+		[DisplayName("Image Path")]
+		[Description("Enter or Select the path to the image file.")]
+		[SampleUsage(@"C:\temp\image.png || {vFilePath} || {ProjectPath}\image.png")]
+		[Remarks("")]
+		[Editor("ShowVariableHelper", typeof(UIAdditionalHelperType))]
+		[Editor("ShowFileSelectionHelper", typeof(UIAdditionalHelperType))]
+		[CompatibleTypes(null, true)]
+		public string v_ImagePath { get; set; }
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs:49]: Please add an image scale property as discussed

**Review by openbots-ff — CHANGES_REQUESTED** [review]

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs (hunk 1 above)
Comment (openbots-aw) [reply OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs]: Done.

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs (hunk 2 above)
Comment (openbots-aw) [reply OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs]: Done.

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs (hunk 3 above)
Comment (openbots-aw) [reply OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelAddImageToCell.cs]: Done.

File: OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelWriteImageToCell.cs [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+		[SampleUsage(@"C:\temp\image.png || {vFilePath} || {ProjectPath}\image.png")]
+		[Remarks("")]
+		[Editor("ShowVariableHelper", typeof(UIAdditionalHelperType))]
+		[Editor("ShowFileSelectionHelper", typeof(UIAdditionalHelperType))]
+		[CompatibleTypes(null, true)]
+		public string v_ImagePath { get; set; }
+
+		[Required]
+		[DisplayName("Image Scale Percentage")]
+		[Description("Enter the scale image percentage to enlarge or reduce the pysical size of image.")]
```
Comment (openbots-ff) [inline OpenBots.Commands/OpenBots.Commands.Microsoft/OpenBots.Commands.Excel/ExcelWriteImageToCell.cs:57]:
```suggestion
		[Description("Enter the image scale percentage to enlarge or reduce the size of the rendered image.")]
```

**Review by openbots-ff — APPROVED** [review]

**Review by openbots-ab — APPROVED** [review]

**Pull request merged**